packages/providers/test/retry.test.ts

```diff
@@ -0,0 +1,138 @@
+import { safeExec } from '@fuel-ts/errors/test-utils';
+
+import Provider from '../src/provider';
+
+// TODO: Figure out a way to import this constant from `@fuel-ts/wallet/configs`
```

Comment:
Is this because of cyclic deps?

Reply:
Yes.

Reply:
It's always helpful to have the reason within the comment.